mne_bids/tests/test_mne_bids.py

```diff
@@ -45,8 +46,29 @@
     shell = True
 
 
-# MEG Tests
-# ---------
+def test_channels_tsv_units():
+    """Test that the original units are assigned if available."""
+    output_path = _TempDir()
+
+    # We use a brainvision testing file to check that raw._orig_units
+    # are assigned as appropriate. First write a channels.tsv
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+    raw = mne.io.read_raw_brainvision(raw_fname)
+    channels_tsv_name = op.join(output_path, 'channels.tsv')
+    _channels_tsv(raw, channels_tsv_name)
+
+    # Now check we have written the correct units
+    df = pd.read_csv(channels_tsv_name, sep='\t', keep_default_na=False)
+    assert df.loc[df['name'] == 'FP1', 'units'].all() == u'µV'
+    assert df.loc[df['name'] == 'CP5', 'units'].all() == 'n/a'
+    assert df.loc[df['name'] == 'CP6', 'units'].all() == u'µS'
+    assert df.loc[df['name'] == 'HL', 'units'].all() == 'n/a'
+    assert df.loc[df['name'] == 'HR', 'units'].all() == 'n/a'
+    assert df.loc[df['name'] == 'Vb', 'units'].all() == 'S'
```

Comment:
same question -- what unit is this?

Reply:
Siemens? These are the original units that are present in the test file ... and they are valid according to BIDS. If you want, I can only check the `n/a` and `µV` units :-)